CONTRIBUTION.md

```diff
@@ -0,0 +1,77 @@
+<h1 align="center">Contributing to The Project </h1>
+
+<p align="center">
+  Welcome to the NotesJar repository! We appreciate your interest in contributing.
+</p>
+
+## Table of Contents
+
+- [Getting Started](#getting-started)
+- [Code Style](#code-style)
+- [Reporting Bugs](#reporting-bugs)
+- [Feature Requests](#feature-requests)
+- [Contribution Process](#contribution-process)
+- [Pull Request Guidelines](#pull-request-guidelines)
+- [License](#license)
+
+## Getting Started
+
+1. Fork the repository.
+2. Clone the forked repository to your local machine:
+    ```bash
+    git clone https://github.com/your-username/[project-name].git 
```

Comment:
```suggestion
    git clone https://github.com/<your-username>/NotesJar.git 
```
Adding project name and also add a line to change `<your-username>` with their own username.